config/servicemappings/edgenetwork.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: core.cnrm.cloud.google.com/v1alpha1
+kind: ServiceMapping
+metadata:
+  name: edgenetwork.cnrm.cloud.google.com
+  namespace: cnrm-system
+spec:
+  name: Edgenetwork
```

Comment:
I think this should be `EdgeNetwork` based on my readings on other servicemapping files. Although I did not find a rule specific for this field. cc @maqiuyujoyce

Reply:
We suggest the service name be the same as the hostname of the prod endpoint of the API following the Pascal Case, and the split of the words (either it's "Edge Network" or "Edgenetwork") should probably follow the branding of the product (and the readability of the name). According to my brief research, there is no occurrences of Edgenetwork in the public doc, so I'd suggest it should be `EdgeNetwork` for service name instead.

Reply:
Done, renamed everything from Edgenetwork -> EdgeNetwork.